### ISSUE TITLE
From a single word, get a list of sentences for usage in assignment type 1

### ISSUE THREAD

**Issue by Spriz** [drunkenflamingos/drunkenflamingo-backend#40]
Could be like "KorpusDK": 

![image](https://cloud.githubusercontent.com/assets/3512268/25782259/77974cd2-3347-11e7-8e25-287a7c059d85.png)

http://ordnet.dk/korpusdk

**Comment by Spriz** [comment]
@PatrickGorgin / Malthe proposed to scrape ordnet.dk - which sure seems doable, they follow some weird pattern with a POST to an URL that returns a HTTP 302 though 🤔

**Comment by Spriz** [comment]
Maybe we should consider to do some sorting on "lix"-level and use a scale like:
```
55 >  Meget svær, faglitteratur på akademisk niveau, lovtekster.
45 - 54 Svær, saglige bøger, populærvidenskabelige værker, akademiske udgivelser.
35 - 44 Middel, dagblade og tidsskrifter.
25 - 34 Let for øvede læsere, ugebladslitteratur og let skønlitteratur for voksne
24 <  Let tekst for alle læsere, børnelitteratur
```
(http://www.nielsgamborg.dk/indhold/lixberegner.htm)

**Comment by PatrickGorgin** [comment]
Not sure it would be necessary since the teacher will be the one to copy paste the text into the app, so they will know to input text of appropriate difficulty levels? :)

**Comment by Spriz** [comment]
Makes sense. I was thinking to use the "KorpusDK" to fetch texts rather thant eacher c/p texts in.

**Comment by PatrickGorgin** [comment]
Yeah, but that would mean that the teacher is potentially limited to what is on KorpusDK, which is a lot, but I don't think they have everything.
